.pre-commit-config.yaml

```diff
@@ -26,7 +33,7 @@ repos:
     - id: debug-statements
       language_version: python3.6
     - id: end-of-file-fixer
-      exclude: ^contracts/contracts/lib/.*\.sol$
+      exclude: ^contracts/contracts/lib/.*\.sol|contracts/contracts/token/TrustlinesNetworkToken.sol$
```

Comment:
why do we exclude the Token contract?

Reply:
The token contract ends with a multi-line comment. prettier formats it in a way that's not compatible with the end-of-file-fixer, because it adds an empty line at the end of the file. Moving the comment to the top of the file would have also fixed it.

see https://github.com/trustlines-protocol/blockchain/pull/234/files#diff-208f0bc21783678041037e523b43ebefR147